### PR TITLE
RPN-1012: description fields update

### PR DESCRIPTION
**Pull request by mazinden**
RPN-1012
RPN-1033

### CODE REVIEW

File: api/protocol-api-nft/openapi.yaml [hunk 1]
```diff
@@ -271,16 +271,19 @@ paths:
     get:
       tags:
         - nft-item-controller
-      description: returns Item by identifier
+      description: Returns NFT by item identifier
```
Comment (evgenynacu) [inline api/protocol-api-nft/openapi.yaml:274]: I think it's better to name it "Returns NFT item by identifier"

File: api/protocol-api-nft/openapi.yaml [hunk 2]
```diff
@@ -427,38 +430,49 @@ paths:
...
           in: query
+          description: "The number of items to return"
           required: false
           schema:
             type: integer
             format: int32
+          example: 100
         - name: showDeleted
           in: query
+          description: "Show deleted items"
```
Comment (evgenynacu) [inline api/protocol-api-nft/openapi.yaml:453]: may be "Include deleted items"?

**Review by evgenynacu — CHANGES_REQUESTED** [review]

File: api/protocol-api-nft/openapi.yaml (hunk 1 above)
Comment (mazinden) [reply api/protocol-api-nft/openapi.yaml:274]: Ok fixed

File: api/protocol-api-nft/openapi.yaml (hunk 2 above)
Comment (mazinden) [reply api/protocol-api-nft/openapi.yaml:453]: yeah that's much better

**Pull request merged**